app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CartsController < ApplicationController
+	before_action :logged_in_user
+	
+  def show
+		@order_items = current_order.order_items
```

Comment:
Really bad indentation here. Just install rubocop and a ruby formatter in vscode.

https://marketplace.visualstudio.com/items?itemName=rebornix.Ruby

> Set ruby.format to rubocop to enable rubocop formatting on save.

>Formatting requires the rubocop gem to be installed. Note that you may have to turn on some of the AutoCorrect functions in your .rubocop.yml file. See the rubocop documentation.